src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -313,12 +301,14 @@ private Solution CreateOrOpenSolution()
 
         private IPersistentStorage GetStorage(Solution solution)
         {
-            var storage = new EsentPersistentStorageService(_persistentEnabledOptionService, testing: true).GetStorage(solution);
+            var storage = GetStorageService().GetStorage(solution);
 
             Assert.NotEqual(NoOpPersistentStorage.Instance, storage);
             return storage;
         }
 
+        protected abstract IPersistentStorageService GetStorageService();
```

Comment:
I'm not seeing that name as being any better.  The name seems to match well with what it does.